circuits/voteverifier/inputs_test.go

```diff
@@ -0,0 +1,176 @@
+package voteverifier
+
+import (
+	"crypto/ecdsa"
+	"crypto/rand"
+	"fmt"
+	"math/big"
+	"os"
+
+	gecdsa "github.com/consensys/gnark-crypto/ecc/secp256k1/ecdsa"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/iden3/go-iden3-crypto/babyjub"
```

Comment:
Do we really need this import if we already have bjj_gnark ?

Reply:
just for the mocked encryption key generation